testsuite/MDAnalysisTests/core/test_universe.py

```diff
@@ -171,6 +171,7 @@ def test_Universe_invalidpermissionfile_IE_msg(self):
         try:
             mda.Universe(os.path.join(temp_dir.name, 'permission.denied.tpr'))
         except IOError as e:
+            #pylint: ignore=unsupported-membership-test
```

Comment:
You can try it on travis. But I doubt that will change the result on travis either. Especially since the marker works just like this for a false positive I'm suppressing in the package folder.